site/content/docs/manifest/backend-service.md

```diff
@@ -134,6 +134,8 @@ count:
   range: 1-10
   cpu_percentage: 70
   memory_percentage: 80
+  requests: 10000
+  response_time: 2s
```

Comment:
I don't think these fields should not be available for a backend service type 😛 -- there is no load balancer

Reply:
good catch!